worker/docker_client.py

```diff
@@ -229,6 +229,35 @@ def download_image(self, docker_image, loop_callback):
                 loop_callback(status)
 
     @wrap_exception('Unable to start Docker container')
+    def run_nvidia_smi(self, args, docker_image):
```

Comment:
I think in the future we're going to deploy workers as docker containers, in which case running `nvidia-smi` inside docker would not require us to have two different versions of worker images (one for gpu, one not) to deploy.